create_book.rb

```diff
@@ -0,0 +1,17 @@
+class CreateBook
+  def initialize(library)
+    @library = library
+    @books = @library.books
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    book = Book.new(title, author)
+    books << book
+    puts ''
+    puts '#=>  Book created successfully'
+  end
+end
```

Comment:
- [ ] As requested by the previous reviewer, the refactor should NOT break the app. The user is able to create a person but the issue is creating the book. Kindly fix this.
Based on this error message, it seems like you're trying to call a method named create_book on an instance of the App class, but this method is not defined within the App class.

Able to create Student 
![image](https://github.com/aima98/OOP-School-Library1/assets/81932478/f04b9013-68d6-4d11-9c96-e1ab0ae52f37)

Able to create a teacher
![image](https://github.com/aima98/OOP-School-Library1/assets/81932478/9a86f422-958f-4c95-877a-8779260e9322)

Error creating a book
![image](https://github.com/aima98/OOP-School-Library1/assets/81932478/9f091ee4-1f2f-436c-a6cb-19314fe0bb52)



### Note that since I cannot check if a rental is being created, ensure everything is working fine before requesting for another review